sunbird-notification/src/test/java/org/sunbird/notification/sms/Message91Test.java

```diff
@@ -2,18 +2,38 @@
 
 import java.util.ArrayList;
 import java.util.List;
+
+import org.apache.http.impl.client.HttpClients;
 import org.junit.Assert;
+import org.junit.BeforeClass;
 import org.junit.FixMethodOrder;
 import org.junit.Test;
+import org.junit.runner.RunWith;
 import org.junit.runners.MethodSorters;
+import org.powermock.core.classloader.annotations.PowerMockIgnore;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
 import org.sunbird.notification.sms.provider.ISmsProvider;
 import org.sunbird.notification.sms.providerimpl.Msg91SmsProvider;
+import org.sunbird.notification.utils.PropertiesCache;
 import org.sunbird.notification.utils.SMSFactory;
 
 /** @author Manzarul */
+
 @FixMethodOrder(MethodSorters.NAME_ASCENDING)
+@RunWith(PowerMockRunner.class)
+@PowerMockIgnore({ "javax.management.*", "javax.net.ssl.*", "javax.security.*" })
+@PrepareForTest({ HttpClients.class ,PropertiesCache.class})
 public class Message91Test {
 
+  @BeforeClass
```

Comment:
Format code changes. Run scripts/setup.sh it auto formats Java code for each file staged for commit to git.